torch/quantization/fake_quantize.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from torch.nn import Module
+from .._jit_internal import weak_module
+from .observer import Observer, observer
+
+@weak_module
+class FakeQuantize(Module):
```

Comment:
should we expose this class in torch/quantization/__init__?

Reply:
yes